api/dictionary_opds.py

```diff
@@ -1,38 +1,54 @@
 from nose.tools import set_trace
 from api.opds.opds_writer import OPDSFeed
+from functools import reduce
+import json
 
 class DictionaryFeed(OPDSFeed):
   SCHEMA_DEFINED_TERM = "http://schema.org/DefinedTerm"
   TIME_FORMAT = '%Y-%m-%dT%H:%M:%SZ%z'
+  SCHEMA_FILE = "api/schema/definition.json"
 
-  def __init__(self, word, url, language, definitions):
-    super().__init__(word, url)
+  def __init__(self, word, url, language, raw_definitions):
+    super().__init__(word, url, self.SCHEMA_FILE)
 
     self.language = language
-    self.feed["definitions"] = []
-
     self.add_metadata()
-    self.add_definitions(definitions)
+    self.definitions = self.update_definitions(raw_definitions)
 
   def add_metadata(self):
     super().add_metadata()
-    self.metadata["title"] = "Definitions for %s" % self.title,
+    self.metadata["title"] = "Definitions for %s" % self.title
     self.metadata["@type"] = self.SCHEMA_DEFINED_TERM
     self.metadata["language"] = self.language
     self.metadata["name"] = self.title
 
   def get_feed(self):
-    return self.feed
+    ns = self.builder.build_classes()
+    definition_builder = ns.Definition
+    definitions = definition_builder(
+      title=self.title,
+      metadata=self.metadata,
+      definitions=self.definitions,
+      links=self.links
+    )
+    # definition["$id"] = self.url
+    # definition["$schema"] = self.JSON_SCHEMA
```

Comment:
These are not required, but suggested for jsonschema. Since we want to use the OPDS 2 spec, this is not needed, right?